net/net-tcp-rpc-client.cpp

```diff
@@ -239,7 +238,7 @@ int tcp_rpcc_parse_execute (struct connection *c) {
       }
       assert (rwm_fetch_lookup (&c->in, &D->packet_len, 4) == 4);
       if (D->packet_len <= 0 || (D->packet_len & 3) || (D->packet_len > TCP_RPCC_FUNC(c)->max_packet_len && TCP_RPCC_FUNC(c)->max_packet_len > 0)) {
-        vkprintf (1, "error while parsing packet: bad packet length %d\n", D->packet_len);
```

Comment:
and the same question here and below. It looks like errors which happen not very often, but when it happens, it's critical. Do I miss something?

Reply:
switch it to `kprintf`